### DISCUSSION TITLE
googlepay

### DISCUSSION THREAD

**Discussion by gonbooster** [creagia/laravel-redsys#36] [Q&A]
¡Buenas noches!

Recientemente estaba explorando la funcionalidad de Redsys para integrar Google Pay (https://pagosonline.redsys.es/googlePay.html), y me preguntaba si sería factible incorporar esta característica a vuestra librería. Confieso que la documentación me ha resultado un tanto compleja de comprender.

¿Sería posible llevar a cabo la implementación de Google Pay en la librería? 

Gracias

**Comment by dtorras** [comment]
Según la documentación, con el método redirección no es necesario ningún cambio técnico. Solo con pedir activarlo a la entidad bancaria aparece como opción de pago.

La "implementación directa" no tengo prevista implementarla en el paquete de momento pero aceptaría una PR con la funcionalidad.

**Comment by gonbooster** [comment]
mmm... vale, es bueno saberlo. Todavía estoy aprendiendo los entresijos de Redsys y por ahora estoy empleando la redirección.

De todas formas, dado que lo estoy empleando en una aplicación react native y me interesa que el diseño sea mas profesional que la propia interfaz que ofrece Redsys, no descarto implementar una "implementación directa".

He visto que para aplicaciones nativas ya tienen documentación pero no se si seria posible replicarlo en una aplicación hibrida.

Si tienes alguna documentación podría investigar y lanzarte una PR en un futuro próximo.
